python/ray/worker.py

```diff
@@ -418,6 +418,17 @@ def put_object(self, object_id, value):
             logger.info(
                 "The object with ID {} already exists in the object store."
                 .format(object_id))
+        except (TypeError, Exception):
+            # It could because the __dict__ of an instance is not serializable
+            # for cloudpickle, typically some cython objects. But the instance
+            # itself could have an `__reduce__` method.
+            register_custom_serializer(type(value), use_pickle=True)
+            warning_message = ("WARNING: Pickling the class {} "
```

Comment:
How about change this warning to `WARNING: Serializing the class {} failed, so are are falling back to cloudpickle..format(type(value))`